tests/test_record_prevalence_healthburden_class.py

```diff
@@ -0,0 +1,161 @@
+import os
+from pathlib import Path
+
+from tlo import Date, Simulation
+from tlo.analysis.utils import parse_log_file
+from tlo.methods import (
+    demography,
+    enhanced_lifestyle,
+    healthburden,
+    healthsystem,
+    mockitis,
+    symptommanager,
+)
+from tlo.methods.fullmodel import fullmodel
+
+resourcefilepath = Path(os.path.dirname(__file__)) / '../resources'
+outputpath = Path("./outputs/")
+
+start_date = Date(2010, 1, 1)
+end_date = Date(2012, 1, 1)
+
+popsize = 1000
+seed = 42
```

Comment:
seed can be a 'magical' kwarg to a function beginning with test and pyyest will populate it for you. (same as tmpdir)